pkg/endpoint/id/id.go

```diff
@@ -95,6 +99,9 @@ func ParseCiliumID(id string) (int64, error) {
 	if err != nil {
 		return 0, fmt.Errorf("invalid numeric cilium id: %s", err)
 	}
+	if n > MaxEndpointId {
+		return 0, fmt.Errorf("endpoint id too large: %d", n)
+	}
```

Comment:
would it make sense to check for `n < 0` as well while we are it?

Reply:
Done.

Reply:
Thanks!